package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "vim-webgl-viewer",
-  "version": "0.5.0",
-  "description": "A 3D model viewer in JavaScript.",
```

Comment:
We will need a description for the package when it is distributed on NPM (https://www.npmjs.com/). Think of NPM as Nuget for JavaScript packages.